scripts/x-charts.exports.json

```diff
@@ -7,11 +7,20 @@
   { "name": "AreaElementPath", "kind": "Variable" },
   { "name": "AreaElementProps", "kind": "TypeAlias" },
   { "name": "AreaPlot", "kind": "Function" },
+  { "name": "AreaPlotProps", "kind": "Interface" },
+  { "name": "AreaPlotSlotComponentProps", "kind": "Interface" },
```

Comment:
I'm not sure about this change. 🙈 
Doing so would technically introduce annoying breaking changes in v6 -> v7, because we renamed all the types to no longer have the `Component` part. 🙈 
Maybe we could backport only a subset of the original changes, mainly including all the component `Props` types? 🤔

Reply:
That is interesting. `AreaPlotSlotComponentProps` doesn't appear on `next`

Reply:
Oh, those got renamed on `next` ... well ... might be best to not cherry pick this specific change then!

Reply:
Yes, exactly, we chose to simplify type names on v7 after we noticed that only one of those was exported in v6. 👍